src/Suave/Cookie.fs

```diff
@@ -42,211 +42,224 @@ module Cookie =
                                                                value = value }
         | "Domain", domain          -> iter + 1, { cookie with domain = Some domain }
         | "Path", path              -> iter + 1, { cookie with path = Some path }
-        | "Expires", expires        -> iter + 1, { cookie with expires = Some (parse_expires expires) }
-        | "HttpOnly", _             -> iter + 1, { cookie with http_only = true }
+        | "Expires", expires        -> iter + 1, { cookie with expires = Some (parseExpires expires) }
+        | "HttpOnly", _             -> iter + 1, { cookie with httpOnly = true }
         | "Secure", _               -> iter + 1, { cookie with secure = true }
         | _                         -> iter + 1, cookie)
-        (0, { HttpCookie.empty with http_only = false }) // default when parsing
+        (0, { HttpCookie.empty with httpOnly = false }) // default when parsing
     |> snd
 
-  module HttpRequest =
+  type HttpRequest with
 
-    let cookies (x : HttpRequest) =
+    member x.cookies =
       x.headers
       |> List.filter (fun (name, _) -> name.Equals "cookie")
-      |> List.flat_map (snd >> parse_cookies)
```

Comment:
"HttpRequest.cookies req" has been deprecated in favour of the property (see ongoing discussion about parsing costs though)